ktxspec.adoc

```diff
@@ -683,6 +690,8 @@ transcode target format is selected.
   components are present. See <<DFD for Supercompressed Data>>.
 
 ===== ZLIB
+* After inflation, the level data follows the uncompressed layout as
+  specified in the <<levelImages>> with <<_mippadding,mipPadding>> omitted.
```

Comment:
Same comment as above though `libktx` does not currently implement zlib compression.

Reply:
Done.